server/user/user.controller.js

```diff
@@ -23,13 +24,18 @@ const inputAttrs = ['email', 'role', 'firstName', 'lastName'];
 const createFilter = q => map(['email', 'firstName', 'lastName'],
   it => ({ [it]: { [Op.iLike]: `%${q}%` } }));
 
+const processOutput = user => {
+  const lastActive = ActivityTracker.lastActive(user.id) || user.lastActive;
```

Comment:
What you did here should in fact go inside `profile` _calculation_. By moving it there you'll reduce number of required code changes :bowtie: